include/mbedtls/hmac_drbg.h

```diff
@@ -204,7 +204,7 @@ int mbedtls_hmac_drbg_seed( mbedtls_hmac_drbg_context *ctx,
                     size_t len );
 
 /**
- * \brief               Initilisation of simpified HMAC_DRBG (never reseeds).
+ * \brief               Initialisation of simplified HMAC_DRBG (never reseeds).
```

Comment:
We've got quite a few typo fixes to "initialize" (US "z" spelling) or variants thereof here; would it make sense to be consistent and change this to `Initialization`? (Obviously there's a whole can of worms there...)

Reply:
We are already inconsistent in UK vs US spellings. This PR doesn't attempt to resolve that one.